### PR TITLE
SSR & Route provider support

### PR DESCRIPTION
**Pull request by mhrrmk**
Test me! 'MASTER'
[Link to SSR-SUPPORT](https://ssr-sup-refine.pankod.com)

Test documentation! 'MASTER'
[Link to SSR-SUPPORT](https://ssr-sup-doc-refine.pankod.com)

### CODE REVIEW

File: packages/core/src/components/antd/datePicker.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Dayjs } from "dayjs";
 import dayjsGenerateConfig from "rc-picker/lib/generate/dayjs";
 import generatePicker from "antd/es/date-picker/generatePicker";
-import "antd/es/date-picker/style/index.css";
+// import "antd/es/date-picker/style/index.css";
```
Comment (mhrrmk) [inline packages/core/src/components/antd/datePicker.tsx:4]: disabled import

File: packages/core/src/components/antd/calendar.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Dayjs } from "dayjs";
 import dayjsGenerateConfig from "rc-picker/lib/generate/dayjs";
 import generateCalendar from "antd/es/calendar/generateCalendar";
-import "antd/es/calendar/style/index.css";
+// import "antd/es/calendar/style/index.css";
```
Comment (mhrrmk) [inline packages/core/src/components/antd/calendar.tsx:4]: disabled import

File: packages/core/src/index.tsx [hunk 3]
```diff
@@ -1,9 +1,8 @@
-import "node_modules/antd/dist/antd.css";
+// import "node_modules/antd/dist/antd.css";
```
Comment (mhrrmk) [inline packages/core/src/index.tsx:1]: disabled import

File: packages/core/src/index.tsx [hunk 4]
```diff
@@ -50,4 +53,4 @@ export { RcFile, UploadFile } from "antd/lib/upload/interface";
 // antd filterDropDownProps (using for <FilterDropDown> component)
 export { FilterDropdownProps } from "antd/lib/table/interface";
 
-export * from "react-query";
+// export * from "react-query";
```
Comment (mhrrmk) [inline packages/core/src/index.tsx:56]: disabled export

File: packages/core/src/hooks/auth/useLogin/index.ts [hunk 5]
```diff
@@ -31,15 +30,15 @@ export const useLogin = <TData, TVariables = {}>(): UseLoginReturnType<
...
+    const { useLocation } = useRouterContext();
+    const { search } = useLocation();
 
     const queryResponse = useMutation<TData, unknown, TVariables, unknown>(
         "useLogin",
         loginFromContext,
         {
             onSuccess: () => {
-                replace(from);
+                replace(search.substring(4) ?? "/");
```
Comment (omeraplak) [inline packages/core/src/hooks/auth/useLogin/index.ts:41]: ups this looks buggy :)

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Pull request merged**